src/app/shared/components/header/header.component.html

```diff
@@ -1,7 +1,16 @@
 <div class="header__logo">
-  <a href="/" title="{{title}}" class="header__logo_link"></a>
+    <a href="/" title="{{title}}" class="header__logo_link"></a>
 </div>
 <div class="header__sighup">
-  <app-login></app-login>
-  <app-logout></app-logout>
+    <div class="login__content">
+        <i class="icon"></i>
+        <a class="login__link" href="/login" *ngIf="!user">Log In</a>
```

Comment:
Do not use 'href' in Angular project. Use 'routerLink' instead and you coud avoid reloading the page